job-utilities/job-details.json

```diff
@@ -37,5 +37,43 @@
                 "year" : "*"
             }
         }
+    },
+    "RefreshHasGranulesOrCwicMap": {
+        "target" : {
+            "endpoint" : "caches/refresh/has-granules-or-cwic-map",
+            "service" : "bootstrap",
+            "single-target" : true,
+            "request-type" : "POST"
+        },
+        "scheduling" : {
+            "type" : "interval",
+            "timing" : {
+                "minutes" : 10,
```

Comment:
Any reason why it's specifically every 1 hour 10 minutes?

Reply:
I wanted 10 minutes after the hour for it to fire.